diff --git a/node_modules/@fluentui/react-theme/lib-commonjs/alias/teamsDark.js b/node_modules/@fluentui/react-theme/lib-commonjs/alias/teamsDark.js
index 34ed6bf..540fbab 100644
--- a/node_modules/@fluentui/react-theme/lib-commonjs/alias/teamsDark.js
+++ b/node_modules/@fluentui/react-theme/lib-commonjs/alias/teamsDark.js
@@ -29,13 +29,13 @@ const generateColorTokens = brand => ({
   colorNeutralForeground4: colors_1.grey[60],
   colorNeutralForegroundDisabled: colors_1.grey[36],
   colorNeutralForegroundInvertedDisabled: colors_1.whiteAlpha[40],
-  colorBrandForegroundLink: brand[90],
+  colorBrandForegroundLink: brand[100],
   colorBrandForegroundLinkHover: brand[110],
-  colorBrandForegroundLinkPressed: brand[100],
-  colorBrandForegroundLinkSelected: brand[90],
-  colorCompoundBrandForeground1: brand[90],
-  colorCompoundBrandForeground1Hover: brand[100],
-  colorCompoundBrandForeground1Pressed: brand[80],
+  colorBrandForegroundLinkPressed: brand[90],
+  colorBrandForegroundLinkSelected: brand[100],
+  colorCompoundBrandForeground1: brand[100],
+  colorCompoundBrandForeground1Hover: brand[110],
+  colorCompoundBrandForeground1Pressed: brand[90],
   colorBrandForeground1: brand[100],
   colorBrandForeground2: brand[110],
   colorNeutralForeground1Static: colors_1.grey[14],
diff --git a/node_modules/@fluentui/react-theme/lib/alias/teamsDark.js b/node_modules/@fluentui/react-theme/lib/alias/teamsDark.js
index 652c0a1..88e5a62 100644
--- a/node_modules/@fluentui/react-theme/lib/alias/teamsDark.js
+++ b/node_modules/@fluentui/react-theme/lib/alias/teamsDark.js
@@ -21,13 +21,13 @@ export const generateColorTokens = brand => ({
   colorNeutralForeground4: grey[60],
   colorNeutralForegroundDisabled: grey[36],
   colorNeutralForegroundInvertedDisabled: whiteAlpha[40],
-  colorBrandForegroundLink: brand[90],
+  colorBrandForegroundLink: brand[100],
   colorBrandForegroundLinkHover: brand[110],
-  colorBrandForegroundLinkPressed: brand[100],
-  colorBrandForegroundLinkSelected: brand[90],
-  colorCompoundBrandForeground1: brand[90],
-  colorCompoundBrandForeground1Hover: brand[100],
-  colorCompoundBrandForeground1Pressed: brand[80],
+  colorBrandForegroundLinkPressed: brand[90],
+  colorBrandForegroundLinkSelected: brand[100],
+  colorCompoundBrandForeground1: brand[100],
+  colorCompoundBrandForeground1Hover: brand[110],
+  colorCompoundBrandForeground1Pressed: brand[90],
   colorBrandForeground1: brand[100],
   colorBrandForeground2: brand[110],
   colorNeutralForeground1Static: grey[14],
