diff --git a/node_modules/@fluentui/react-northstar/dist/commonjs/themes/teams-dark-v2/siteVariables.js b/node_modules/@fluentui/react-northstar/dist/commonjs/themes/teams-dark-v2/siteVariables.js
index 7845336..8585905 100644
--- a/node_modules/@fluentui/react-northstar/dist/commonjs/themes/teams-dark-v2/siteVariables.js
+++ b/node_modules/@fluentui/react-northstar/dist/commonjs/themes/teams-dark-v2/siteVariables.js
@@ -72,7 +72,7 @@ var colorScheme = {
     foreground2: colors.brand['450'],
     foreground3: colors.brand['200'],
     foreground4: colors.white,
-    border: colors.grey['450'],
+    border: colors.grey['475'],
     border1: colors.brand['800'],
     border2: colors.brand['800'],
     foregroundHover: colors.brand['450'],
@@ -91,7 +91,7 @@ var colorScheme = {
     foregroundActive2: colors.brand['50'],
     backgroundActive: colors.brand['450'],
     backgroundActive1: colors.brand['450'],
-    borderActive: colors.grey['450'],
+    borderActive: colors.grey['430'],
     borderActive1: colors.brand['800'],
     borderActive2: colors.brand['800'],
     foregroundFocus: colors.brand['450'],
diff --git a/node_modules/@fluentui/react-northstar/dist/commonjs/themes/teams-high-contrast/colors.js b/node_modules/@fluentui/react-northstar/dist/commonjs/themes/teams-high-contrast/colors.js
index 35a8eb6..7a892e0 100644
--- a/node_modules/@fluentui/react-northstar/dist/commonjs/themes/teams-high-contrast/colors.js
+++ b/node_modules/@fluentui/react-northstar/dist/commonjs/themes/teams-high-contrast/colors.js
@@ -7,7 +7,7 @@ var _siteVariables = require("../teams/siteVariables");
 
 var _colors = require("../teams/colors");
 
-var accessibleYellow = '#ffff01';
+var accessibleYellow = '#ffff00';
 exports.accessibleYellow = accessibleYellow;
 var accessibleGreen = '#3ff23f'; // always disabled color in high contrast
 
diff --git a/node_modules/@fluentui/react-northstar/dist/commonjs/themes/teams-v2/siteVariables.js b/node_modules/@fluentui/react-northstar/dist/commonjs/themes/teams-v2/siteVariables.js
index ca79811..a553f64 100644
--- a/node_modules/@fluentui/react-northstar/dist/commonjs/themes/teams-v2/siteVariables.js
+++ b/node_modules/@fluentui/react-northstar/dist/commonjs/themes/teams-v2/siteVariables.js
@@ -26,6 +26,7 @@ var colors = {
     440: '#666',
     450: '#616161',
     460: '#5c5c5c',
+    475: '#525252',
     500: '#424242',
     550: '#3d3d3d',
     600: '#333',
@@ -40,16 +41,16 @@ var colors = {
   },
   brand: {
     50: '#e8ebfa',
-    100: '#dcdffa',
+    100: '#dce0fa',
     200: '#c5cbfa',
-    300: '#acb3fa',
-    400: '#9399f5',
-    450: '#9399f5',
-    500: '#7a80eb',
+    300: '#9299f7',
+    400: '#7f85f5',
+    450: '#7f85f5',
+    500: '#7579eb',
     600: '#5b5fc7',
-    700: '#444791',
-    800: '#3d3e78',
-    900: '#383966',
+    700: '#4f52b2',
+    800: '#444791',
+    900: '#3d3e78',
     1000: '#2f2f4a'
   }
 };
@@ -111,7 +112,7 @@ var colorScheme = {
     background4: colors.brand['800'],
     background5: colors.brand['100'],
     foreground: colors.brand['600'],
-    foreground1: colors.brand['600'],
+    foreground1: colors.brand['700'],
     foreground2: colors.brand['700'],
     foreground3: colors.brand['200'],
     foreground4: colors.white,
@@ -134,7 +135,7 @@ var colorScheme = {
     foregroundActive2: colors.brand['50'],
     backgroundActive: colors.brand['600'],
     backgroundActive1: colors.brand['600'],
-    borderActive: colors.grey['200'],
+    borderActive: colors.grey['270'],
     borderActive1: colors.brand['50'],
     borderActive2: colors.brand['300'],
     foregroundFocus: colors.brand['600'],
diff --git a/node_modules/@fluentui/react-northstar/dist/dts/src/themes/teams-dark-v2/siteVariables.js b/node_modules/@fluentui/react-northstar/dist/dts/src/themes/teams-dark-v2/siteVariables.js
index ddec8c0..f9c1555 100644
--- a/node_modules/@fluentui/react-northstar/dist/dts/src/themes/teams-dark-v2/siteVariables.js
+++ b/node_modules/@fluentui/react-northstar/dist/dts/src/themes/teams-dark-v2/siteVariables.js
@@ -67,7 +67,7 @@ exports.colorScheme = {
         foreground2: exports.colors.brand['450'],
         foreground3: exports.colors.brand['200'],
         foreground4: exports.colors.white,
-        border: exports.colors.grey['450'],
+        border: exports.colors.grey['475'],
         border1: exports.colors.brand['800'],
         border2: exports.colors.brand['800'],
         foregroundHover: exports.colors.brand['450'],
@@ -86,7 +86,7 @@ exports.colorScheme = {
         foregroundActive2: exports.colors.brand['50'],
         backgroundActive: exports.colors.brand['450'],
         backgroundActive1: exports.colors.brand['450'],
-        borderActive: exports.colors.grey['450'],
+        borderActive: exports.colors.grey['430'],
         borderActive1: exports.colors.brand['800'],
         borderActive2: exports.colors.brand['800'],
         foregroundFocus: exports.colors.brand['450'],
diff --git a/node_modules/@fluentui/react-northstar/dist/dts/src/themes/teams-v2/siteVariables.d.ts b/node_modules/@fluentui/react-northstar/dist/dts/src/themes/teams-v2/siteVariables.d.ts
index f525322..a0ec028 100644
--- a/node_modules/@fluentui/react-northstar/dist/dts/src/themes/teams-v2/siteVariables.d.ts
+++ b/node_modules/@fluentui/react-northstar/dist/dts/src/themes/teams-v2/siteVariables.d.ts
@@ -19,6 +19,7 @@ export declare const colors: {
         440: string;
         450: string;
         460: string;
+        475: string;
         500: string;
         550: string;
         600: string;
diff --git a/node_modules/@fluentui/react-northstar/dist/dts/src/themes/teams-v2/siteVariables.js b/node_modules/@fluentui/react-northstar/dist/dts/src/themes/teams-v2/siteVariables.js
index 9924f47..c18e4ab 100644
--- a/node_modules/@fluentui/react-northstar/dist/dts/src/themes/teams-v2/siteVariables.js
+++ b/node_modules/@fluentui/react-northstar/dist/dts/src/themes/teams-v2/siteVariables.js
@@ -23,6 +23,7 @@ exports.colors = {
         440: '#666',
         450: '#616161',
         460: '#5c5c5c',
+        475: '#525252',
         500: '#424242',
         550: '#3d3d3d',
         600: '#333',
diff --git a/node_modules/@fluentui/react-northstar/dist/es/themes/teams-dark-v2/siteVariables.js b/node_modules/@fluentui/react-northstar/dist/es/themes/teams-dark-v2/siteVariables.js
index 0d235f4..4f38161 100644
--- a/node_modules/@fluentui/react-northstar/dist/es/themes/teams-dark-v2/siteVariables.js
+++ b/node_modules/@fluentui/react-northstar/dist/es/themes/teams-dark-v2/siteVariables.js
@@ -64,7 +64,7 @@ export var colorScheme = {
     foreground2: colors.brand['450'],
     foreground3: colors.brand['200'],
     foreground4: colors.white,
-    border: colors.grey['450'],
+    border: colors.grey['475'],
     border1: colors.brand['800'],
     border2: colors.brand['800'],
     foregroundHover: colors.brand['450'],
@@ -83,7 +83,7 @@ export var colorScheme = {
     foregroundActive2: colors.brand['50'],
     backgroundActive: colors.brand['450'],
     backgroundActive1: colors.brand['450'],
-    borderActive: colors.grey['450'],
+    borderActive: colors.grey['430'],
     borderActive1: colors.brand['800'],
     borderActive2: colors.brand['800'],
     foregroundFocus: colors.brand['450'],
diff --git a/node_modules/@fluentui/react-northstar/dist/es/themes/teams-high-contrast/colors.js b/node_modules/@fluentui/react-northstar/dist/es/themes/teams-high-contrast/colors.js
index ecc72aa..10dc2cd 100644
--- a/node_modules/@fluentui/react-northstar/dist/es/themes/teams-high-contrast/colors.js
+++ b/node_modules/@fluentui/react-northstar/dist/es/themes/teams-high-contrast/colors.js
@@ -1,6 +1,6 @@
 import { colors } from '../teams/siteVariables';
 import { createColorScheme as createEmptyColorScheme } from '../teams/colors';
-export var accessibleYellow = '#ffff01';
+export var accessibleYellow = '#ffff00';
 export var accessibleGreen = '#3ff23f'; // always disabled color in high contrast
 
 export var accessibleCyan = '#1aebff';
diff --git a/node_modules/@fluentui/react-northstar/dist/es/themes/teams-v2/siteVariables.js b/node_modules/@fluentui/react-northstar/dist/es/themes/teams-v2/siteVariables.js
index e70da20..cbbf693 100644
--- a/node_modules/@fluentui/react-northstar/dist/es/themes/teams-v2/siteVariables.js
+++ b/node_modules/@fluentui/react-northstar/dist/es/themes/teams-v2/siteVariables.js
@@ -20,6 +20,7 @@ export var colors = {
     440: '#666',
     450: '#616161',
     460: '#5c5c5c',
+    475: '#525252',
     500: '#424242',
     550: '#3d3d3d',
     600: '#333',
@@ -34,16 +35,16 @@ export var colors = {
   },
   brand: {
     50: '#e8ebfa',
-    100: '#dcdffa',
+    100: '#dce0fa',
     200: '#c5cbfa',
-    300: '#acb3fa',
-    400: '#9399f5',
-    450: '#9399f5',
-    500: '#7a80eb',
+    300: '#9299f7',
+    400: '#7f85f5',
+    450: '#7f85f5',
+    500: '#7579eb',
     600: '#5b5fc7',
-    700: '#444791',
-    800: '#3d3e78',
-    900: '#383966',
+    700: '#4f52b2',
+    800: '#444791',
+    900: '#3d3e78',
     1000: '#2f2f4a'
   }
 };
@@ -104,7 +105,7 @@ export var colorScheme = {
     background4: colors.brand['800'],
     background5: colors.brand['100'],
     foreground: colors.brand['600'],
-    foreground1: colors.brand['600'],
+    foreground1: colors.brand['700'],
     foreground2: colors.brand['700'],
     foreground3: colors.brand['200'],
     foreground4: colors.white,
@@ -127,7 +128,7 @@ export var colorScheme = {
     foregroundActive2: colors.brand['50'],
     backgroundActive: colors.brand['600'],
     backgroundActive1: colors.brand['600'],
-    borderActive: colors.grey['200'],
+    borderActive: colors.grey['270'],
     borderActive1: colors.brand['50'],
     borderActive2: colors.brand['300'],
     foregroundFocus: colors.brand['600'],
