diff --git a/node_modules/@fluentui/react-northstar/dist/es/themes/teams-dark-v2/siteVariables.js b/node_modules/@fluentui/react-northstar/dist/es/themes/teams-dark-v2/siteVariables.js
index 4f35eb6..2110164 100644
--- a/node_modules/@fluentui/react-northstar/dist/es/themes/teams-dark-v2/siteVariables.js
+++ b/node_modules/@fluentui/react-northstar/dist/es/themes/teams-dark-v2/siteVariables.js
@@ -20,7 +20,7 @@ export var colorScheme = {
     background5: colors.grey['600'],
     background6: colors.grey['910'],
     background7: colors.grey['850'],
-    border: colors.grey['450'],
+    border: colors.grey['440'],
     border1: colors.grey['850'],
     border2: colors.grey['900'],
     border3: colors.grey['550'],
@@ -29,22 +29,22 @@ export var colorScheme = {
     foregroundHover1: colors.white,
     foregroundHover2: colors.white,
     backgroundHover: colors.grey['550'],
-    backgroundHover1: colors.grey['750'],
+    backgroundHover1: colors.grey['700'],
     backgroundHover2: 'transparent',
     backgroundHover3: colors.grey['650'],
     backgroundHover4: colors.grey['700'],
     backgroundHover6: colors.grey['550'],
-    borderHover: colors.grey['430'],
+    borderHover: colors.grey['420'],
     foregroundPressed: colors.white,
-    backgroundPressed: colors.grey['650'],
+    backgroundPressed: colors.grey['800'],
     backgroundPressed2: colors.grey['850'],
     backgroundPressed3: colors.grey['800'],
     foregroundActive: colors.white,
     foregroundActive1: colors.white,
     backgroundActive: colors.grey['600'],
-    backgroundActive1: colors.grey['800'],
+    backgroundActive1: colors.grey['750'],
     backgroundActive2: colors.grey['575'],
-    borderActive: colors.grey['440'],
+    borderActive: colors.grey['430'],
     // foregroundFocus: not specified,
     foregroundFocus1: colors.grey['220'],
     foregroundFocus2: colors.grey['220'],
@@ -58,7 +58,7 @@ export var colorScheme = {
     foregroundDisabled1: colors.grey['460'],
     borderDisabled: colors.grey['500'],
     backgroundDisabled: colors.grey['800'],
-    backgroundDisabled1: colors.grey['800']
+    backgroundDisabled1: colors.grey['870']
   },
   brand: {
     background: colors.brand['700'],
diff --git a/node_modules/@fluentui/react-northstar/dist/es/themes/teams-v2/siteVariables.js b/node_modules/@fluentui/react-northstar/dist/es/themes/teams-v2/siteVariables.js
index 3feb280..e0a77a0 100644
--- a/node_modules/@fluentui/react-northstar/dist/es/themes/teams-v2/siteVariables.js
+++ b/node_modules/@fluentui/react-northstar/dist/es/themes/teams-v2/siteVariables.js
@@ -17,6 +17,7 @@ export var colors = {
     310: '#adadad',
     350: '#949494',
     400: '#8a8a8a',
+    420: '#757575',
     430: '#707070',
     440: '#666',
     450: '#616161',
@@ -102,7 +103,7 @@ export var colorScheme = {
     borderFocus: colors.black,
     borderFocusWithin: colors.white,
     foregroundDisabled: colors.grey['250'],
-    foregroundDisabled1: colors.grey['250'],
+    foregroundDisabled1: colors.grey['270'],
     borderDisabled: colors.grey['200'],
     backgroundDisabled: colors.grey['100'],
     backgroundDisabled1: colors.grey['100']
